### PR TITLE
[WIP] Plug kinto storage

### PR DESCRIPTION
**Pull request by almet**
Add the ability to save the generated form in Kinto

- [x] Save the form a kinto instance;
- [x] Redirect the user once the form has been created, display links to admin and form;
- [x] Create a view where users can fill in the form;
- [x] List all the results in a specific route;
- [x] Be able to configure the kinto server to use (in the config for now);
- [x] Separate the concerns of the form creation and the rest;
- [x] Fix the redirection when submitting new records;
- [ ] [Add tests](https://github.com/Kinto/formbuilder/issues/26);

### CODE REVIEW

File: formbuilder/routes.js [hunk 1]
```diff
@@ -26,15 +30,25 @@ const LinkBack = () => {
...
-    <Route path="settings"
-      components={{...common, fieldList: LinkBack, content: FormOptionsContainer}} />
-    <Route path="json"
-      components={{...common, fieldList: LinkBack, content: JsonViewContainer}} />
-    <Route path="*" components={{
-      fieldList: FieldListContainer,
-      content: _ => <h1>Page not found.</h1>
-    }}/>
-  </Route>
+  <div>
```
Comment (n1k0) [inline formbuilder/routes.js]: Why this div?

**Comment by n1k0** [comment]
Looks sharp!

**Comment by almet** [comment]
@n1k0 I wonder what's the actual best way to do the configuration here.

I've seen that in kinto-admin there is a `config/config.json` file and that the client is doing a `fetch` of this file. I would like, if possible, to avoid having to read the file over HTTP at startup. Do you see any way to do this?

File: formbuilder/routes.js (hunk 1 above)
Comment (almet) [inline formbuilder/routes.js]: removed!

**Comment by n1k0** [comment]
> I wonder what's the actual best way to do the configuration here.
> I would like, if possible, to avoid having to read the file over HTTP at startup

There's a static config for the formbuilder [already](https://github.com/Kinto/formbuilder/blob/master/formbuilder/config.js), what more would you need?

**Comment by almet** [comment]
I was thinking about a way to start the formbuilder with a `--config` option pointing to the correct `json` configuration (and not JS), but that's probably for another issue, I will use the existing `config.js` for now.

**Comment by n1k0** [comment]
> a way to start the formbuilder with a --config option

Remember that the formbuilde ris a webpage and has no clue how it's served (and that's a feature) :)

**Comment by almet** [comment]
Moving forward (discussed with @n1k0).

**Pull request merged**